datamover/pkg/drivers/https/prepare.go

```diff
@@ -63,19 +63,19 @@ func getLocationInfo(ctx context.Context, j *flowtype.Job, in *pb.RunJobRequest)
 
 func refreshSrcLocation(ctx context.Context, obj *osdss3.Object, srcLoca *LocationInfo, destLoca *LocationInfo,
 	locMap map[string]*LocationInfo) (newSrcLoca *LocationInfo, err error) {
-	if obj.Backend != srcLoca.BakendName && obj.Backend != "" {
+	if obj.Location != srcLoca.BakendName && obj.Location != "" {
```

Comment:
we are comparing obj.Location to srcLoca.BakendName ??

Reply:
In OSDS, the location of an object is its corresponding backend.  We changed the name from backend to location so it is compatible with AWS s3.